idpol/idpol.py

```diff
@@ -10,7 +10,12 @@ def ruleset1(words):
     for word in words.split(' '):
         if word != 'die':
             word = word.replace('ie','ee')
-        word = word.replace('ble','bur')
+        if not word.startswith('ble'):
```

Comment:
so lets say we have table, right?
table -> tabur

but under this we get something like
tablecloth -> tabrecroth

i would think about what actually makes this "bur" sound. first idea is that it's like a vowel before the ble. like "invisible" "soluble"